### ISSUE TITLE
Not Scraping Data Correctly for NSP

### ISSUE THREAD

**Issue by PSXDupe** [gibaBR/Switch-Backup-Manager#134]
SBM 1.2.1 is not scraping game title or developer correctly, developer is blank and game title is that of the filename.

Probably the Nintendo Website has change structure :)

**Comment by Sovikos** [comment]
Yeah I've had this problem too, and wondering if it's the titlekeys or keys file?

**Comment by deadair3210** [comment]
The only time I've had the program fail to name a file is when the file is damaged or altered in some way incorrectly. The program pulls game and update files' titles from the files themselves, so if the file is malformed or missing things, then it falls back to the filename
